README.md

```diff
@@ -18,5 +18,31 @@ WP2Static is an open source project, maintained by many generous developers over
 
 ## [Contributors](https://wp2static.com/contributors)
 
+If you want to do a contribution to project, please, follow next instructions:
+
+1. Fork project with button in top of WP2static github [home page](https://github.com/WP2Static/wp2static)
+2. Clone your project to your development computer (please, change <your-account> by your account name):
+<br/>``git clone https://github.com/<your-account>/wp2static.git``
```

Comment:
I used to encounter more issues with https vs SSH when cloning and working. Do you think using SSH for remote is easier for a new user or either is OK? Perhaps add both lines as options here?

Reply:
I think _https_ is easier for new users. _SSH_ is more profesional and pretty sure if someone wants to change to _SSH_ will be easier. I mean if you are pro you know how to switch :)

Reply:
Good point!